targets/python36-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile

```diff
@@ -236,6 +236,12 @@ PermitEmptyPasswords yes\n" > /etc/ssh/sshd_config
 # ssh port
 EXPOSE 22
 
+# ==================================================================
+# Documentation notebook
+# ------------------------------------------------------------------
+RUN mkdir -p /var/notebooks
+COPY files/var/notebooks/Storage.ipynb /var/notebooks
```

Comment:
I'm slightly lost here. I thought that we agreed that this file is copied to /var/storage (if there is no one there yet), but I don't see where does it happen.

Reply:
It's copied to the image and then, when user runs Jupyter from web UI `platform-pipelines` copy the file to storage.

Reply:
Ah, OK, thanks.